### PR TITLE
BEACD 55 | Added list page for VAT

### PR DESCRIPTION
**Pull request by kurtlogan**

### CODE REVIEW

File: test/views/vat/WhichVATServicesToAddViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+  val serviceInfoContent = HtmlFormat.empty
+
+  def createView = () => whichVATServicesToAdd(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  def createViewUsingForm =
+    (form: Form[_]) => whichVATServicesToAdd(frontendAppConfig, form)(serviceInfoContent)(fakeRequest, messages)
+
+  "WhichVATServicesToAdd view" must {
+    behave like normalPage(createView, messageKeyPrefix)
```
Comment (gaurav-hmrc) [inline test/views/vat/WhichVATServicesToAddViewSpec.scala]: add content tests

File: test/views/vat/RegisteredForVATECSalesViewSpec.scala [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  val serviceInfoContent = HtmlFormat.empty
+
+  def createView =
+    () => registeredForVAT(frontendAppConfig, form, viewAction)(serviceInfoContent)(fakeRequest, messages)
+
+  def createViewUsingForm =
+    (form: Form[_]) => registeredForVAT(frontendAppConfig, form, viewAction)(serviceInfoContent)(fakeRequest, messages)
+
+  "RegisteredForVATECSales view" must {
+    behave like normalPage(createView, messageKeyPrefix)
```
Comment (gaurav-hmrc) [inline test/views/vat/RegisteredForVATECSalesViewSpec.scala]: add content test

File: test/views/vat/WhichVATServicesToAddViewSpec.scala (hunk 1 above)
Comment (kurtlogan) [reply test/views/vat/WhichVATServicesToAddViewSpec.scala]: The only content on the page is the header and radio buttons, both have tests already.

File: test/views/vat/RegisteredForVATECSalesViewSpec.scala (hunk 2 above)
Comment (kurtlogan) [reply test/views/vat/RegisteredForVATECSalesViewSpec.scala]: The only content on the page is the header and radio buttons, both have tests

**Pull request merged**